src/index.ts

```diff
@@ -1,5 +1,5 @@
 // TODO: Remove request-permission-polyfil before going to production
-import "./request-permission-polyfil";
+//import "./request-permission-polyfil";
```

Comment:
why?

Reply:
either remove it or uncomment it

Reply:
My suggestion is to remove this line instead of commenting.

Reply:
I don't need it for now